code/modules/vehicles/cars/clowncar.dm

```diff
@@ -9,16 +9,32 @@
 	movedelay = 0.6
 	car_traits = CAN_KIDNAP
 	key_type = /obj/item/bikehorn
+	light_system = MOVABLE_LIGHT_DIRECTIONAL
+	light_range = 8
+	light_power = 2
+	light_on = FALSE
+	var/headlight_colors = list(COLOR_RED, COLOR_ORANGE, COLOR_YELLOW, COLOR_LIME, COLOR_BRIGHT_BLUE, COLOR_CYAN, COLOR_PURPLE)
 	var/droppingoil = FALSE
 	var/RTDcooldown = 150
 	var/lastRTDtime = 0
 	var/thankscount
 	var/cannonmode = FALSE
 	var/cannonbusy = FALSE
 
+/obj/vehicle/sealed/car/clowncar/Initialize()
+	. = ..()
+	START_PROCESSING(SSobj,src)
+
+/obj/vehicle/sealed/car/clowncar/process()
+	if(light_on && obj_flags & EMAGGED )
```

Comment:
```suggestion
	if(light_on && (obj_flags & EMAGGED))
```

Reply:
My concerns more with the trailing space, but I just like parentheses around bitops.